source/_docs/guides/drupal-8-simplesamlphp.md

```diff
@@ -0,0 +1,264 @@
+---
+title: How to Implement SimpleSAMLphp as idP for Drupal 8 on Pantheon 
+description: Setting up an identity provider on Drupal 8 using SimpleSAMLphp.
+draft: true
+tags: [code, drupal-8]
+categories: [drupal, developing]
+contributors: Javi-er
+---
+
+There are a number of articles detailing how to implement the SAML 2.0 protocol, but if you want to host your site on Pantheon and use Drupal 8 as your identity provider, there are a few details to take in account.
+<div class="alert alert-info">
+<h4 class="info">Note</h4><p markdown="1">The following tutorial assumes that you are on a linux based computer or a Mac. Also that your project is running on Apache.
+</p>
+</div>
+
+## Before You Begin
+
+Before starting with this guide, make sure you have the following:
+
+ - Have a working Drupal 8 site.
+ - Download the latest version of [SimpleSAMLphp](https://simplesamlphp.org/download).
```

Comment:
Where does this need to be installed?

Reply:
Just added the instructions where to extract it, those were lower in the document.